test/Libraries/DynamoPythonTests/PythonEvalTestsWithLibraries.cs

```diff
@@ -196,17 +206,87 @@ import clr
 typedDictionary = DummyCollection.AcceptDictionary(d)
 typedDictionary['four'] = 4
 
-OUT = untypedDictionary, typedDictionary
+# Python dict => .NET IEnumerable - Returns keys in both engines
+sortedKeys = List.Sort(d)
+
+OUT = untypedDictionary, typedDictionary, sortedKeys
 ";
             var empty = new ArrayList();
-            var expected = new Dictionary<string, int> {
+            var expectedDictionary = new Dictionary<string, int> {
                 { "one", 1 }, { "two", 2 }, { "three", 3 }, { "four", 4 }
             };
+            var expectedKeys = new List<string> { "one", "three", "two" };
```

Comment:
Is it because `List` unordered so sequence does not matter here?

Reply:
The order is alphabetic